### DISCUSSION TITLE
Possiblity to consider Stock list

### DISCUSSION THREAD

**Discussion by stockie77** [TandoorRecipes/recipes#1340] [Ideas]
Hello,
is it possible  if i create a shopping list that the system consider the Stock list in my house?
Or does anybody have  a idea how i can do that / program it?

**Comment by smilerz** [comment]
Do you mean don’t add items you already have on hand?

If so, that will be implemented with the next release.

**Comment by stockie77** [comment]
i have a list/table with all Articles e.g 200 g sugar,2000 g flour, 10 tomatos.....
and if i create a shopping list based on a recipes the system calculate with the article on the stock and displayed the source location 
of already exist Article in the list and the article what i must buy in the supermarket

**Comment by smilerz** [comment]
No. The extent of the functionality is what I described above.  Full inventory may be available in the future.
